eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdout.java

```diff
@@ -44,12 +43,6 @@
  */
 @XmirObject(oname = "stdout")
 public class EOstdout extends PhDefault {
-
-    /**
-     * Default out.
-     */
-    private static final PrintStream OUT = System.out;
```

Comment:
@maxonfjvipon Why did you remove this field?

Reply:
@Graur 
1. `System.out` is already `static final`
2. For some reason if we set `System.out` to the variable (like `OUT`) the tests with `@StdIo` annotation and `StdOut` argument won't work